libnavui-dropin/src/main/res/values/styles.xml

```diff
@@ -1,85 +1,192 @@
+        <item name="tripProgressViewBackgroundColor">@color/colorSurface</item>
+    </style>
+
+    <style name="DropInTripProgressViewTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">@color/colorOnSurface</item>
         <item name="android:textSize">14sp</item>
     </style>
 
-    <style name="MapboxStyleExpandableButton">
-        <item name="android:elevation">0dp</item>
-        <item name="android:background">@drawable/mapbox_bg_button</item>
-        <item name="extendableButtonIconTint">@color/mapbox_button_fg_color</item>
-        <item name="extendableButtonTextAppearance">@style/MapboxStyleExpandableButton.TextAppearance</item>
+    <!-- Speed limit view styling -->
+    <style name="DropInStyleSpeedLimit" parent="MapboxStyleSpeedLimit">
+        <item name="speedLimitTextColor">@color/colorOnSurface</item>
+        <item name="speedLimitMutcdBorderColor">@color/colorOnSecondaryVariant</item>
+        <item name="speedLimitBackgroundColor">@color/colorSurface</item>
+        <item name="speedLimitViennaBorderColor">@color/mapbox_speed_limit_view_vienna_border</item>
     </style>
-    <style name="MapboxStyleExpandableButton.TextAppearance">
-        <item name="android:textColor">@color/mapbox_button_fg_color</item>
+
+    <style name="DropInSpeedLimitTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">@color/colorOnSurface</item>
         <item name="android:textSize">14sp</item>
     </style>
 
-    <style name="MapboxStyleExpandableButton.Primary">
-        <item name="android:background">@drawable/mapbox_bg_button_primary</item>
-        <item name="extendableButtonIconTint">@color/mapbox_button_primary_fg_color</item>
-        <item name="extendableButtonTextAppearance">@style/MapboxStyleExpandableButton.Primary.TextAppearance</item>
+    <!-- Road name view styling -->
+    <style name="DropInRoadNameViewTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">@color/colorOnSurface</item>
+        <item name="android:textSize">14sp</item>
     </style>
-    <style name="MapboxStyleExpandableButton.Primary.TextAppearance">
-        <item name="android:textColor">@color/mapbox_button_primary_fg_color</item>
+
+    <!-- Drag handle styling -->
+    <style name="DropInStyleDragHandle" parent="">
+        <item name="android:layout_width">48dp</item>
+        <item name="android:layout_height">5dp</item>
+        <item name="android:layout_marginTop">10dp</item>
+        <item name="android:background">@drawable/mapbox_bg_handle</item>
+    </style>
+
+    <!-- Info panel styling -->
+    <style name="DropInInfoPanelHeadlineTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textColor">@color/colorOnSurface</item>
+        <item name="android:textStyle">normal</item>
+        <item name="android:textSize">27sp</item>
```

Comment:
nit: seems like this should reference another dimension

e.g., @dimen/mapbox_text_size_headline

Reply:
similar to other textSize styles

Reply:
Done.